mlir/test/Integration/Dialect/SparseTensor/GPU/CUDA/lit.local.cfg

```diff
@@ -1,2 +1,4 @@
 if not config.enable_cuda_runner or not config.mlir_run_cuda_sm80_tests:
     config.unsupported = True
+
+config.substitutions.append(("%format", config.gpu_compilation_format))
```

Comment:
can we please use a slightly more specific name for this (format is very generic, how about at least gpu_format or so)